lib/iris/fileformats/grib/_message.py

```diff
@@ -376,7 +376,7 @@ def _get_key_value(self, key):
                        'satelliteNumber', 'instrumentType',
                        'scaleFactorOfCentralWaveNumber',
                        'scaledValueOfCentralWaveNumber',
-                       'longitudes', 'latitudes', 'distinctLatitudes')
+                       'longitudes', 'latitudes')
```

Comment:
> Are longitudes and latitudes computed or coded keys?

This distinction appears to be template dependent.  Due to bugs (and perhaps other factors)   it is also grib api version dependent.

So, using grib api 1.14:
 - in GDT4 & GDT5, latitudes are reported as coded;
   - this is true, there is an array of numbers
 - in GDt40 & GDT41 latitudes are report as computed;
   - this seems true, these are worked out

I originally included an exception report is 'get_computed_key' retrieved a coded key, but this was grib api version intolerant so I removed it.

At version 1.14, this approach appears to mandate that the new get_computed_key function is used whenever a gribapi computation is used.
This seems like a safe, but not so gribapi version critical interpretation for our use